### PR TITLE
[4.0]namespace administrator components com_fields

### PR DESCRIPTION
**Pull request by brianteeman**
continuing the work to namespace joomla4 one admin component at a time.

please wait for drone to successfully complete before reviewing.

### CODE REVIEW

File: administrator/components/com_fields/tmpl/field/edit.php [hunk 1]
```diff
@@ -8,28 +8,33 @@
  */
 defined('_JEXEC') or die;
 
+use Joomla\CMS\HTML\HTMLHelper;
```
Comment (Quy) [inline administrator/components/com_fields/tmpl/field/edit.php]: Remove duplicate

File: administrator/components/com_fields/tmpl/fields/default_batch_body.php [hunk 2]
```diff
@@ -8,9 +8,12 @@
  */
 defined('_JEXEC') or die;
 
+use Joomla\CMS\HTML\HTMLHelper;
```
Comment (Quy) [inline administrator/components/com_fields/tmpl/fields/default_batch_body.php]: Remove duplicate

File: administrator/components/com_fields/tmpl/field/edit.php (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_fields/tmpl/field/edit.php]: done

File: administrator/components/com_fields/tmpl/fields/default_batch_body.php (hunk 2 above)
Comment (brianteeman) [reply administrator/components/com_fields/tmpl/fields/default_batch_body.php]: done

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**